src/RouteRouter.js

```diff
@@ -20,54 +20,73 @@ class RouteRouter extends AbstractRouter {
         let end: string = req.query.end;
         let arriveBy: boolean = req.query.arriveBy == '1'
         let departureTimeQuery: string = req.query.time;
-        let departureTimeNowMs = parseFloat(departureTimeQuery) * 1000 - 600000;
-        let departureTimeFifteenMinutesLater = departureTimeNowMs + 900000;
+        let departureTimeNowMs = parseFloat(departureTimeQuery) * 1000;
+        //let departureTimeFifteenMinutesLater = departureTimeNowMs + 900000;
         let departureTimeDateNow = new Date(departureTimeNowMs).toISOString();
-        let departureTimeDateLater = new Date(departureTimeFifteenMinutesLater).toISOString();
+        //let departureTimeDateLater = new Date(departureTimeFifteenMinutesLater).toISOString();
         
         try {
             let parameters: any = {
                 vehicle: "pt",
                 weighting: "short_fastest",
                 point: [start, end],
-                points_encoded: false
+                points_encoded: false,
             };
             parameters["pt.arrive_by"] = arriveBy;
-           // parameters["ch.disable"] = true;
+            parameters["ch.disable"] = true;
 
-            // if this was set to > 3.0, sometimes the route would suggest getting off busy earlier and walk half a mile instead of waiting longer
+            // if this was set to > 3.0, sometimes the route would suggest getting off bus earlier and walk half a mile instead of waiting longer
             parameters["pt.walk_speed"] = 3.0;
             parameters["pt.earliest_departure_time"] = departureTimeDateNow;
-
+            parameters["pt.profile"] = true;
+            parameters["pt.limit_solutions"] = 6
+            
             let routeNowReq: any = axios.get('http://localhost:8988/route', {
                 params: parameters,
                 paramsSerializer: (params: any) => qs.stringify(params, { arrayFormat: 'repeat' })
             });
 
-            parameters["pt.earliest_departure_time"] = departureTimeDateLater;
-            let routeLaterReq: any = axios.get('http://localhost:8988/route', {
-                params: parameters,
-                paramsSerializer: (params: any) => qs.stringify(params, { arrayFormat: 'repeat' })
-            });
+            // let paramForLater = Object.assign({}, parameters);
+            // paramForLater["pt.earliest_departure_time"] = departureTimeDateLater;
+            // let routeLaterReq: any = axios.get('http://localhost:8988/route', {
+            //     params: paramForLater,
+            //     paramsSerializer: (params: any) => qs.stringify(params, { arrayFormat: 'repeat' })
+            // });
 
             //Wait until all requests finish
-            let [routeNowResult, routeLaterResult] = await Promise.all([routeNowReq, routeLaterReq]);
-            
+            let [routeNowResult] = await Promise.all([routeNowReq]);
```

Comment:
are you planning on making a second request? if so, i think it's okay to leave the Promise.all()